src/components/home/Login.tsx

```diff
@@ -1,12 +1,86 @@
-import { useState } from "react";
-import { LoadingButton } from "@mui/lab";
+import React from "react";
+import {
+    Container, Box, TextField,
+    Button
+} from "@mui/material";
 
 export default function Login() {
-	const [loading, setLoading] = useState(false);
+    const [email, SetEmail] = React.useState("");
+    const [password, SetPassword] = React.useState("");
+    const [emailError, setEmailError] = React.useState(false);
+    const [passwordError, setPasswordError] = React.useState(false);
 
-	return (
-		<>
-			<LoadingButton variant="contained" onClick={() => setLoading(true)} loading={loading} fullWidth>Login</LoadingButton>
-		</>
-	);
+    //Submit button checking if email and password have valid values, setting error if not
+    const handleSubmit = (e: React.MouseEvent<HTMLButtonElement>) => {
+        e.preventDefault();
+
+        if (email === "" || email === null) {
+            setEmailError(true);
+        }
+        if (password === "" || password === null) {
+            setPasswordError(true);
+        }
+    }
+
+    //onChange functions setting email and password values + clearing any existing errors
+    const handleEmail = (e: any) => {
+        SetEmail(e.target.value);
+
+        if(emailError === true && email != "") {
+            setEmailError(false);
+        }
+    }
+    
+    const handlePassword = (e: any) => {
+        SetPassword(e.target.value);
+
+        if(passwordError === true && password != "") {
+            setPasswordError(false);
+        }
+    }
+
+    return (
+        <Container sx={{
+            display: "flex", flexDirection: "column", alignItems: "center",
+            justifyContent: "center", height: "100%"
+        }}>
+
+            <h2>Login</h2>
+
+            {/*Form component containing text fields + submit button*/}
+            <Box
```

Comment:
Here's another case of where you can use a Stack component instead of a Box with explicit display: flex; styling.